internal_worker_base.go

```diff
@@ -53,6 +53,7 @@ type (
 		asyncActivityClient
 		workflowTimerClient
 		SideEffect(f func() ([]byte, error), callback resultHandler)
+		GetVersion(changeID string, maxSupported, minSupported Version) Version
```

Comment:
The order of variables is inversed. minSupported, maxSupported. Can you add unit test for this layer as well?